pythermalcomfort/charts.py

```diff
@@ -0,0 +1,624 @@
+from math import ceil, floor
+import numpy as np
+import plotly.graph_objects as go
+from plotly.subplots import make_subplots
+from scipy.optimize import brentq
+
+from pythermalcomfort.models import pmv
+from pythermalcomfort.utilities import v_relative, clo_dynamic, psy_ta_rh
+from pythermalcomfort.theme import (
+    tight_margins,
+    index_mapping_dictionary,
+)
+
+
+def adaptive_chart(df):
```

Comment:
we should discuss whether this df should already contains the results of the adaptive model. so we can simplify significantly the code below.